packages/icons-react/tasks/build.js

```diff
@@ -183,7 +184,10 @@ function convertToJSX(node) {
  */
 function formatAttributes(attrs) {
   return Object.keys(attrs).reduce((acc, key, index) => {
-    const attribute = `${key}="${attrs[key]}"`;
+    const attribute = key.startsWith('data-')
```

Comment:
Any chance `key` may contain `aria-` attributes?

Reply:
@asudoh most likely not, but wouldn't hurt to throw this in there and just have a denylist of things we don't want to transform 👍 Are there any others we'd want to include?

Reply:
Just pushed up a quick change for this, feel free to add on to it!

Reply:
From https://reactjs.org/blog/2017/09/08/dom-attributes-in-react-16.html, the safest way seems to be checking if the attribute is one of [the known ones](https://reactjs.org/docs/dom-elements.html#all-supported-html-attributes). If React library exports that list we can use it, but otherwise, seems that `aria-` and `data-` seems enough, assuming non-data custom attribute use case is rare.

Reply:
Yeah, thankfully in this case we control the data so don't have to worry about user-supplied stuff 👍 Most of the time the `key` will be a `d` attribute for `<path>`, but sometimes we'll have `fill-rule` (or other SVG attributes) alongside `data-inner-path`.